source/extensions/filters/network/zookeeper_proxy/decoder.cc

```diff
@@ -617,7 +617,7 @@ void DecoderImpl::decode(Buffer::Instance& data, DecodeType dtype, uint64_t full
 }
 
 void DecoderImpl::parseConnectResponse(Buffer::Instance& data, uint64_t& offset, uint32_t len,
-                                       const std::chrono::milliseconds& latency) {
+                                       const std::chrono::milliseconds latency) {
```

Comment:
why all these changes (const ref --> const)?

Is that related to the functional changes you are making? Maybe you could peel those off to a separate PR?

Reply:
copying small object is preferred for perf at least most of the repo is in favors of copy chrono type. Hence, I asked to remove ref while he introduce new function with ref to `std::chrono::milliseconds`. Yeah small diff is preferred but I thought consistency might be another thing to consider.

Reply:
consistency is fine but why don't break that out into a separate PR. It's not related to this one.

Reply:
this PR have new functions with the chrono types so splitting forces us to either
(a) bring more `const std::chrono::milliseconds&` as , or  
(b) make inconsistency inside zk filter to have both ref and copying

Another PR is fine I agree so I won't insist

Reply:
Thank you for the discussion!
I will revert unrelated changes and create another PR after this one.